### DISCUSSION TITLE
announce: lexer Nelua for CudaText editor

### DISCUSSION THREAD

**Discussion by Alexey-T** [edubart/nelua-lang#199] [Show and tell]
Free text editor CudaText has new lexer Nelua.
To install lexer: menu "Plugins / Addons Manager / Install", find "Nelua" item.

It was tested on many files from Nelua Github repo.

![nelua](https://user-images.githubusercontent.com/4154978/197172016-799cf9c1-40f8-46dc-aef4-fb1e9ec5cd4d.png)

**Comment by edubart** [comment]
Nice to see more editor support for Nelua :)
